lldb/test/API/commands/command/script/TestCommandScript.py

```diff
@@ -161,6 +161,19 @@ def cleanup():
         )
         self.expect("my_command", substrs=["a.out"])
 
+        # Test that without --overwrite we are not allowed to redefine the command.
+        self.expect(
+            "command script add my_command --class welcome.TargetnameCommand",
+            substrs=[
+                (
+                    'user command "my_command" already exists and force replace was'
+                    " not set by --overwrite or 'settings set"
+                    " interpreter.require-overwrite false'"
+                ),
+            ],
+            error=True,
```

Comment:
this is the format I got from the `black` tool...

Reply:
I welcome our new robot formatting overlords, even if they do strange things sometimes. Iirc it uses `"` if there is `'` in the string and vice versa, instead of `"""`.